docs/bnb_greenfield/faq/general-faqs.md

```diff
@@ -0,0 +1,86 @@
+---
+title: General FAQ
+description: A list of BNB Greenfield frequently asks questions.  
+keywords: [BNB Greenfield token, BNB Greenfield usage]
+---
+
```

Comment:
I think general faq and storage faqs can be merged into one, as they are quite similar.

Reply:
Done